src/test/resources/schema_test.sql

```diff
@@ -1,16 +1,34 @@
 CREATE TABLE  IF NOT EXISTS users (
-                       user_id SERIAL PRIMARY KEY,
-                       first_name VARCHAR(50) NOT NULL,
-                       last_name VARCHAR(50) NOT NULL,
-                       pan VARCHAR(16) UNIQUE NOT NULL,
-                       cvv VARCHAR(3) NOT NULL
-);
+                                      user_id SERIAL PRIMARY KEY,
```

Comment:
form sql queries like this
CREATE TABLE  IF NOT EXISTS users (
user_id      SERIAL PRIMARY KEY,
first_name VARCHAR(50) NOT NULL,
last_name  VARCHAR(50) NOT NULL,
pan            VARCHAR(16) UNIQUE NOT NULL,
cvv            VARCHAR(3) NOT NULL
);